src/containers/GamesPage/index.jsx

```diff
@@ -2,11 +2,19 @@ import React from "react";
 import GamesPageImageGrid from "../../components/gamespage/GamesPageImageGrid";
 import BottomCurve from "../../components/BottomCurve";
 
+export const gameData = {
```

Comment:
when you are providing test data, you want it to be in the test only. 

as a general rule, we never have test data in production code

the point is not whether your component works on "real data", its whether it works for "any gamedata of valid format i provide it"

basically, move this as MOCK_GAME_DATA into your test.js

and here, you should be providing the real data, not the fake data.

Reply:
PTAL. I also believe that not passing props to test component was the problem with memory game tests. Because invariant function was trying to find length of undefined. That undefined props was being passed from parent.

Reply:
I will create an issue for it